hasher-matcher-actioner/hmalib/lambdas/api/content.py

```diff
@@ -37,6 +38,14 @@ def to_json(self) -> t.Dict:
         return asdict(self)
 
 
+@dataclass
+class ImageResponse(JSONifiable):
```

Comment:
nit: if it works for video why call it `ImageResponse` ?

Reply:
(if it is because of the path of the endpoint we can just updated that as well yeah?)

Reply:
Yeah. That makes more sense. I was being lazy. I'll make the change. Thanks!